AlfrescoApp/View Controllers/Settings View Controller/SettingsViewController.m

```diff
@@ -182,75 +184,155 @@ - (void)handleActionWithPreferenceIdentifier:(NSString *)preferenceIdentifier
+            [confirmation show];
+        }
+    }];
+}
+
+- (void) sendFeedbackHandler
+{
+    if ([MFMailComposeViewController canSendMail])
+    {
+        MFMailComposeViewController *emailController = [[MFMailComposeViewController alloc] init];
+        emailController.mailComposeDelegate = self;
+        [emailController setSubject:[self getFeedbackSubject]];
+        [emailController setToRecipients:@[kSettingsSendFeedbackAlfrescoRecipient]];
         
-        [resetAccountAlert showWithCompletionBlock:^(NSUInteger buttonIndex, BOOL isCancelButton) {
-            if (!isCancelButton)
-            {
-                // Reset accounts, delete cache databases, tmp folder, downloads
-                // Remove accounts
-                [[AccountManager sharedManager] removeAllAccounts];
-                // Delete cache
-                CoreDataCacheHelper *cacheHelper = [[CoreDataCacheHelper alloc] init];
-                [cacheHelper removeAllAvatarDataInManagedObjectContext:nil];
-                [cacheHelper removeAllDocLibImageDataInManagedObjectContext:nil];
-                [cacheHelper removeAllDocumentPreviewImageDataInManagedObjectContext:nil];
-                // Remove downloads
-                [[DownloadManager sharedManager] removeAllDownloads];
-                // Remove all contents of the temp folder
-                [[AlfrescoFileManager sharedManager] clearTemporaryDirectory];
-                
-                [[AnalyticsManager sharedManager] trackEventWithCategory:kAnalyticsEventCategorySettings
-                                                                  action:kAnalyticsEventActionClearData
-                                                                   label:kAnalyticsEventLabelFull
-                                                                   value:@1];
-                
-                UIAlertView *confirmation = [[UIAlertView alloc] initWithTitle:NSLocalizedString(@"settings.reset.confirmation.title", @"Reset Complete Title")
-                                                                       message:NSLocalizedString(@"settings.reset.confirmation.message", @"Reset Complete Message")
-                                                                      delegate:self
-                                                             cancelButtonTitle:NSLocalizedString(@"Close", @"Close")
-                                                             otherButtonTitles:nil];
-                [confirmation show];
-            }
-        }];
+        // Content body template
+        NSString *footer = [self getFeedbackFooter];
+        NSString *messageBody = [NSString stringWithFormat:@"<br><br>%@", footer];
+        [emailController setMessageBody:messageBody isHTML:YES];
+        emailController.modalPresentationStyle = UIModalPresentationPageSheet;
+        
+        [self presentViewController:emailController animated:YES completion:nil];
     }
```

Comment:
There should be an `else {}` block here handling the case where the user does not have an email account configured; otherwise the action fails without any indication as to why. We have an example of that already in `ActionViewHandler.m`